### PR TITLE
Thirdperson toggle and console variable interface

### PR DESCRIPTION
**Pull request by Doctor-Coomer**
Hello, this set of patches binds thirdperson to left shift, and adds the interface for console variables.

### CODE REVIEW

File: source_sdk/cvar/convar/convar.h [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#include <stdbool.h>
+#include <stdint.h>
+
+typedef void * convar;
```
Comment (faluthe) [inline source_sdk/cvar/convar/convar.h:4]: Interesting. I like using `void *` for simplicity and learning purposes but maybe I'll start using this throughout the project for clarity.

File: hooks/create_move/create_move.c [hunk 2]
```diff
@@ -5,6 +5,8 @@
 #include "../../source_sdk/user_cmd.h"
 #include "../paint_traverse/paint_traverse.h"
 #include "create_move.h"
+#include "../../source_sdk/cvar/convar/convar.h"
+#include "../../source_sdk/cvar/cvar.h"
```
Comment (faluthe) [inline hooks/create_move/create_move.c]:
```suggestion
#include "../../config/config.h"
#include "../../utils/utils.h"
#include "../../source_sdk/cvar/convar/convar.h"
#include "../../source_sdk/cvar/cvar.h"
#include "../../source_sdk/engine_client/engine_client.h"
#include "../../source_sdk/entity/entity.h"
#include "../../source_sdk/user_cmd.h"
#include "../paint_traverse/paint_traverse.h"
#include "create_move.h"
```

Comment (faluthe) [reply hooks/create_move/create_move.c]: Alphabetize

Comment (Doctor-Coomer) [reply hooks/create_move/create_move.c]: :+1:

File: source_sdk/cvar/convar/convar.h (hunk 1 above)
Comment (Doctor-Coomer) [reply source_sdk/cvar/convar/convar.h:4]: Maybe there could also be type definitions for entities? I'd like to respect what you want in your code when I'm publishing to your repository, but `void *` can be a little abstract to what its pointing to.

**Pull request merged**